bitcoin/src/amount.rs

```diff
@@ -517,6 +517,22 @@ impl Amount {
     #[deprecated(since = "0.31.0", note = "Use Self::MIN instead")]
     pub const fn min_value() -> Amount { Amount(u64::min_value()) }
 
+    /// Convert from a value expressing integer values of bitcoins to an [Amount] in const context.
```

Comment:
Should have `# Panics` section.

Reply:
Lucky we have @Kixunil on review to catch the things that I miss :)

Reply:
Thanks, I added a panic section